src/TableContent.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import URI from 'urijs'
+
+import tableHeaderCells from './tableHeaderCells'
+
+const TableContent = ({enableIndex, tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex,
+  ascendingOrder, enableDownload, checkedRows, currentPageData, host, entriesPerPage, currentPage,
+  tableHeaderOnClick, tableHeaderOnChange, downloadOnChange}) =>
+  <div className={`row expanded`}>
+    <div className={`small-12 columns`} >
+      <Table border>
+        <Table.Head>
+          {
+            enableIndex && <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={100} flexGrow={100}>Index</Table.TextHeaderCell>
+          }
+
+          {
+            tableHeaderCells(tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex, ascendingOrder,
+              columnNumber => tableHeaderOnClick(columnNumber),
+              (value, columnNumber) => tableHeaderOnChange(value, columnNumber)
+            )
+          }
+
+          {
+            enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
+              {
+                checkedRows.length > 0 ?
+                  <a href={URI(`${host}experiments/download/zip`).search({accession: checkedRows})}>
+              Download {checkedRows.length} {checkedRows.length === 1 ? `entry` : `entries`}</a> :
+                  `Download`
+              }
+            </Table.TextHeaderCell>
+          }
+        </Table.Head>
+
+        <Table.Body>
+          {currentPageData.map((data, index) => {
+            return (
+              <Table.Row height={`auto`} backgroundColor={index % 2 === 0 ? `white`:`#F5F6F7`} paddingY={14} key={`row${index}`}>
```

Comment:
Use colour `#F1F1F1`, following [EBI framework pattern](https://ebiwd.github.io/EBI-Pattern-library/patterns/tables/#component-detail). Also, a hover effect would also be a good touch.